recipes/snakeobjects/meta.yaml

```
@@ -0,0 +1,48 @@
+package:
+  name: snakeobjects
+  {% set version = "3.0" %}
+
+source:
+  url:  https://github.com/iossifovlab/snakeobjects/archive/{{ version }}.tar.gz
+  sha256: 0429266b45689a46e52ab66343d58cfa1d3d3f0eb9f026543b2ceb636d5fef8c
+
+
+build:
+  # If the installation is complex, or different between Unix and Windows, use
+  # separate bld.bat and build.sh files instead of this key.  Add the line
+  # "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or
+  # "skip: True  # [not win]" to limit to Windows.
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed -vv .
+  noarch: python
+  number: 0
+  entry_points:
+    # {% for entry in data['entry_points']['console_scripts'] %}
+    # - {{ entry.split('=')[0].strip() }} = {{ entry.split('=')[1].strip() }}
+    # {% endfor %}
+    - sobjects = snakeobjects.cli:cli
+  
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - snakemake
+
+test:
+  imports:
+    - snakeobjects
+  commands:
+    - sobjects --help
+
+
+about:
+  home: https://github.com/iossifovlab/snakeobjects
+  summary: Snakeobjects, an object-oriented workflow management system based on snakemake
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  maintainers:
+   - Boris Yamrom
```

```suggestion
   - yamrom
```
better to use a github username, then you will get a notification if any changes are made to the recipe